src/components/thumbnail/tab-panel-documents-section.tsx

```diff
@@ -79,25 +79,27 @@ function getSectionDocs(section: NavTabSectionModelType, stores: IStores, classS
   return sectDocs;
 }
 
-export const TabPanelDocumentsSection = observer(({ tab, section, index, numOfSections, stores, scale, selectedDocument,
+export const TabPanelDocumentsSection = observer(({ tab, section, index, numSections, stores, scale, selectedDocument,
```

Comment:
_If_ it were the case that `numSections` could be `undefined` (which apparently it can't) then you could say `numSections = 0` right here and that will initialize it to 0 if and only if it's undefined, otherwise it will leave it alone. It operates like a default argument. That's what I was thinking when I made the original recommendation.